config/loader.go

```diff
@@ -10,7 +10,7 @@ const (
 	EnvDealbotHome = "DEALBOT_HOME"
 
 	// DefaultDaemonListenAddr is a host:port value, where we set up an HTTP endpoint.
-	DefaultDaemonListenAddr = "localhost:8765"
+	DefaultDaemonListenAddr = "localhost:0"
```

Comment:
Changing to :0 for now, so that I can easily run a few of these locally while testing.